tasks/fh-build.js

```diff
@@ -403,14 +382,43 @@ module.exports = function(grunt) {
         'shell:fh-run-array:integrate_cover', 'shell:fh-run-array:accept_cover',
         'shell:fh-report-cov:lcov', 'shell:fh-report-cov:cobertura'
       ]);
-    } else if (this.target === 'analysis') {
-      grunt.task.run(['plato:fh']);
-    } else if (this.target === 'shrinkwrap') {
-      grunt.task.run(['shell:fh-run-array:fhshrinkwrap']);
     } else if (this.target === 'default') {
-      grunt.task.run(['jshint', 'fh:test', 'fh:dist']);
+      grunt.task.run(['eslint', 'fh:test', 'fh:dist']);
     } else {
       grunt.fail.warn('Unknown target provided to `grunt fh`');
     }
   });
+
```

Comment:
Thanks for the feedback @wtrocki, I agree :)

I will probably remove the `fh-dist-no-bundle-deps` and `fh-dist-only-bundle-deps` before merging, because it turns out that that's not how the parameters are passed to them.